privilege/privileges/cache.go

```diff
@@ -263,6 +271,66 @@ func (p *MySQLPrivilege) FindAllRole(activeRoles []*auth.RoleIdentity) []*auth.R
 	return ret
 }
 
+func (p *MySQLPrivilege) IncLoginFail(user, host string) int {
+	key := user + "@" + host
+	cnt, exist := p.PwdErrorCnt[key]
```

Comment:
maybe need to take care this map should be concurrent access and have race condition both in map itself and map value